README.md

```diff
@@ -6,12 +6,15 @@ Inter UI is a typeface specially designed for user interfaces.
 
 ### [⬇︎ Download the latest release](https://github.com/rsms/inter/releases)
 
+### Windows and Mac Users
 After downloading the zip from above:
 
 1. Double-click the downloaded zip file to unpack or open it.
 2. Follow the instructions in "install-mac.txt" or "install-win.txt", depending
    on what operating system you're using.
 
+### Linux Users
+For Arch Linux users, [a package is available in the AUR](https://aur.archlinux.org/packages/ttf-inter-ui/).
```

Comment:
Please add general instructions for Linux (I'm only using Linux without a window manager, so I'm not familiar with font management in various dists; perhaps there's no unified way. If so, please add a note pointing that out or somehow make it clear to other Linux users that they might have to go someplace else to learn how to install fonts.)

Reply:
I don't know what general instructions for Linux look like.  Almost every distribution is different here.   Feel free to close out this PR.